### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by scottcwilson** [zencart/zencart#6860] [Announcements]
Zen Cart Version 2.1.0. 

Release Notes: https://docs.zen-cart.com/release/whatsnew_2.1.0/

Known Bugs: https://www.zen-cart.com/showthread.php?230454-Known-Bugs-in-2-1-0

Release Specific Upgrade Considerations: https://docs.zen-cart.com/user/upgrading/release_specific_upgrade_considerations/#zen-cart-210

<hr /><em>This discussion was created from the release <a href='https://github.com/zencart/zencart/releases/tag/v2.1.0'>v2.1.0 </a>.</em>

**Comment by bilbaxo** [comment]
big work
thank you at all
J.Green - ios.fr

**Comment by torvista** [comment]
Thanks for your enduring efforts.
